OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +30,53 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+
+		[Desc("Space between lines as a percentage (default) of line height or fixed pixel amount.")]
```

Comment:
Should they be left for when that does happen, converted to comments, or removed?